view/theme/frio/config.php

```diff
@@ -57,6 +57,7 @@ function theme_content(App $a)
 	$arr = [];
 
 	$arr['scheme']           = PConfig::get(local_user(), 'frio', 'scheme', PConfig::get(local_user(), 'frio', 'schema'));
+    $arr['share_string']     = '';
```

Comment:
Can you replace the spaces with a tab here?

Reply:
tehtud